stock_picking_invoice_validation/model/sale_order.py

```diff
@@ -8,17 +8,24 @@
 #    coded by: Hugo Adan, hugo@vauxoo.com
 ############################################################################
 
-from openerp import models, fields
+from openerp import models, fields, api
 
 
 class SaleOrder(models.Model):
     _inherit = 'sale.order'
 
+    @api.multi
+    def _get_wh_conf(self):
+        key = "stock.check_inv_pick"
+        check_inv_pick = self.env["ir.config_parameter"].get_param(
+            key)
+        return check_inv_pick
+
```

Comment:
Hello.

Functional checks like this one can not be in ir.config.parameter, they must have its entries in config setting in the warehous section, please check that.